api/common/__init__.py

```diff
@@ -150,21 +147,24 @@ def call(self,
             req = Request(url, data=data, headers=headers, method="POST")
 
         elif call_type == APIType.DELETE:
-            print(f"Api.DELETE === {url} -- -H {headers} -D {data}")
             data = json.dumps(data).encode("utf-8")
             req = Request(url, data=data, headers=headers, method="DELETE")
 
         elif call_type == APIType.PATCH:
-            print(f"Api.PATCH === {url} -- -H {headers} -D {data}")
             data = json.dumps(data).encode("utf-8")
             req = Request(url, data=data, headers=headers, method="PATCH")
 
         elif call_type == APIType.PUT:
-            print(f"Api.PUT == {url} -- -H {headers} -D {data}")
             data = json.dumps(data).encode("utf-8")
             req = Request(url, data=data, headers=headers, method="PUT")
         else:
             print(f"Unknown === {url}")
+            debug(f"Unknown === {url}")
+
+        print(f"Sending HTTP request: {req.method} {req.get_full_url()}\r\nHeaders: {req.headers}\r\nData: {req.data}")
```

Comment:
do we need duplicate information ? I understand that we can ditch print, or convert it to debug but both print same data :-)

Reply:
We could get rid of prints, but prints do not appear in kodi.log unless set debug mode in Kodi, not plugin settings